net-dns/blocky/blocky-9999.ebuild

```diff
@@ -0,0 +1,94 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit fcaps go-module systemd shell-completion
+
+DESCRIPTION="Fast and lightweight DNS proxy as ad-blocker with many features written in Go"
+HOMEPAGE="https://github.com/0xERR0R/blocky/"
+
+DOCUMENTATION_COMMIT=9c6a86eb163e758686c5d6d4d5259deb086a8aa9
+
+if [[ ${PV} == 9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/0xERR0R/blocky.git"
+else
+	SRC_URI="
+	https://github.com/0xERR0R/blocky/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/rahilarious/gentoo-distfiles/releases/download/${P}/deps.tar.xz -> ${P}-deps.tar.xz
+	doc? ( https://github.com/0xERR0R/blocky/archive/${DOCUMENTATION_COMMIT}.tar.gz -> ${P}-docs.tar.gz )
+"
+	KEYWORDS="~amd64"
+fi
+
+# main
+LICENSE="Apache-2.0"
+# deps
+LICENSE+=" AGPL-3 BSD-2 BSD ISC MIT MPL-2.0"
+SLOT="0"
+IUSE="doc"
+
+# TODO: will do later
+RESTRICT="test"
```

Comment:
Why not now? Especially as you're not putting this into guru...

Reply:
I just tried now, on your say. But some failed tests are causing `die`d ebuild. I even tried it with `go` (instead of ego) without `|| die`